addons/website/controllers/main.py

```diff
@@ -849,6 +877,55 @@ def theme_customize_bundle_reload(self):
     # Server actions
     # ------------------------------------------------------
 
+    @http.route(['/website/theme_upload_font'], type='json', auth='user', website=True)
+    def theme_upload_font(self, name, mimetype, data):
```

Comment:
You never use the mimetype in the method (you recreate it from the file's extension).